ts/components/ui/Switch.tsx

```diff
@@ -12,14 +13,18 @@ const maybeDisabled = fromPredicate(
  * NativeBase Switch component styled with the app's brand primary color
  */
 export default class Switch extends React.Component<NativeBase.Switch> {
-  public render() {
+  public render(accessibleLabelText) {
```

Comment:
The `render` function doesn't accept any argument: it should use `this.props` and `this.state` as the only input sources. This is also the reason we the build is failing.

Please see also [the official documentation](https://reactjs.org/docs/react-component.html#render).